tasks/tasks.py

```diff
@@ -230,3 +230,50 @@ def run_pytest(self):
                 '--html=/vagrant/report.html'
                 ).format(test_suite=self.test_suite)],
                 timeout=None))
+
+
+class ToxTests(JobTask):
+    action_name = 'tox'
+
+    def __init__(self, template, git_repo,
+                 timeout=constants.RUN_PYTEST_TIMEOUT, **kwargs):
+        super(ToxTests, self).__init__(template, timeout=timeout, **kwargs)
+        self.git_repo = git_repo
+        self.template_name = template['name']
+        self.template_version = template['version']
+
+    def _before(self):
```

Comment:
duplicated code; entire `_before()` method can be omitted, it's already done in the method implemented in the parent